### DISCUSSION TITLE
Python 3.11 Support

### DISCUSSION THREAD

**Discussion by Vezzp** [facebookresearch/pytorch3d#1558] [General]
Hi! Any plans for providing Anaconda builds for Python 3.11?

**Comment by bottler** [comment]
We were caught out on this by the PyTorch 2.0.1 release, for which we released PyTorch3D 0.7.4. Normally I wouldn't expect PyTorch bug fix releases to change the dependency versions, but PyTorch 2.0.1 introduces builds for Python 3.11, so ideally we should have builds for that. I expect we will soon.

**Reply by Vezzp** [reply]
Sounds great!
